polkadot/runtime/westend/constants/src/lib.rs

```diff
@@ -128,6 +128,7 @@ pub mod xcm {
 		const ROOT_INDEX: u32 = 0;
 		// The bodies corresponding to the Polkadot OpenGov Origins.
 		pub const FELLOWSHIP_ADMIN_INDEX: u32 = 1;
+		#[deprecated = "use `xcm::latest::BodyId::Treasury` instead"]
```

Comment:
this will be in a separate PR, after I get merged this - https://github.com/polkadot-fellows/runtimes/pull/149